akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -223,8 +223,10 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
    *
    * Note that for large clusters it is more efficient to use `InitialStateAsSnapshot`.
    */
-  @varargs def subscribe(subscriber: ActorRef, initialStateMode: SubscriptionInitialStateMode, to: Class[_]*): Unit =
+  @varargs def subscribe(subscriber: ActorRef, initialStateMode: SubscriptionInitialStateMode, to: Class[_]*): Unit = {
+    require(to.length > 0, "at least one `ClusterDomainEvent` class is required")
```

Comment:
great suggestion, that actually revealed a "bug" in ClusterMetricsDisabledSpec
see 7fbaaa110445df21de2671e58f20d6fdc6a2f5ab